ui-v2/app/templates/dc/services/show.hbs

```diff
@@ -7,7 +7,13 @@
     {{#block-slot 'header'}}
       <h1>
         {{ item.Service.Service }}
-        <span data-test-external-source="{{service/external-source item.Service}}" style={{{ concat 'background-image: ' (css-var (concat '--' (service/external-source item.Service) '-color-svg') 'none')}}} data-tooltip="Registered via {{service/external-source item.Service}}">Registered via {{service/external-source item.Service}}</span>
+{{#with (service/external-source item.Service) as |externalSource| }}
+  {{#with (css-var (concat '--' externalSource '-color-svg') 'none') as |bg| }}
```

Comment:
Just a heads up `with` will be replace with `let` when you upgrade (I _think_ it's deprecated in more recent versions).